zboxcore/sdk/common.go

```diff
@@ -6,15 +6,395 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"os"
+	"strings"
 	"sync"
+	"testing"
 	"time"
 
 	"github.com/0chain/gosdk/zboxcore/blockchain"
 	"github.com/0chain/gosdk/zboxcore/fileref"
 	. "github.com/0chain/gosdk/zboxcore/logger"
+	"github.com/0chain/gosdk/zboxcore/sdk/mocks"
 	"github.com/0chain/gosdk/zboxcore/zboxutil"
+	"github.com/stretchr/testify/assert"
+
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	configDir            = "test"
+	syncTestDir          = configDir + "/" + "sync"
+	syncDir              = syncTestDir + "/" + "sync_alloc"
+	textPlainContentType = "text/plain"
 )
 
+func blobberIDMask(idx int) string {
```

Comment:
If the changes added here on `common.go` are gonna be used for unit testing purposes only, perhaps put split it instead and add as `common_test.go`.

Reply:
Yes, I gree.